release-pipeline.yml

```
@@ -0,0 +1,1956 @@
+              sudo ls -lRt $(root_dir)/
+              cp $(root_dir)/azcopy_linux* $(Build.ArtifactStagingDirectory)
+            displayName: 'List the binaries and add to staging directory'
+
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: 'azCopy-binaries'
+            displayName: 'Publish binaries'
+            condition: succeededOrFailed()
+          
+          - script: |
+              rm -rf $(Build.ArtifactStagingDirectory)/azcopy*
+            displayName: "Remove binaries from staging directory"
+
+          - script: |
+              export GOARCH=amd64
+              export GOOS=linux
+              # Run azcopy --version and capture the output
+              azcopyOutput=$($(root_dir)/azcopy_linux_amd64 --version)
+
+              # Extract the version part
+              if echo "$azcopyOutput" | grep -q "azcopy version"; then
+                version=$(echo "$azcopyOutput" | sed -n 's/^azcopy version \([0-9]*\.[0-9]*\.[0-9]*[^ ]*\).*/\1/p')
+                if [[ -n "$version" ]]; then
+                  echo "Extracted version: $version"
+                  
+                  # Set the pipeline variable (for Azure DevOps)
+                  echo "##vso[task.setvariable variable=azcopy_version]$version"
+                else
+                  echo "Unable to extract version from azcopy output"
+                  exit 1
+                fi
+              else
+                echo "azcopy version string not found in output"
+                exit 1
+              fi
+            displayName: 'Verify and extract the generated build version'
+
+          - script: |
+              linux_amd64_dir="$(work_dir)/azcopy_linux_amd64_$(azcopy_version)"
+              echo "##vso[task.setvariable variable=linux_amd64_dir]$linux_amd64_dir"
+
+              linux_se_amd64_dir="$(work_dir)/azcopy_linux_se_amd64_$(azcopy_version)"
+              echo "##vso[task.setvariable variable=linux_se_amd64_dir]$linux_se_amd64_dir"
+
+              mkdir -p $linux_amd64_dir
+              mkdir -p $linux_se_amd64_dir
+              mkdir -p $(archives)
+              ls -ltR $(work_dir)
+            displayName: 'Create required directories'
+
+          - script: |
+              mkdir -p pkgDir/usr/bin/
+              mv $(root_dir)/azcopy_linux_amd64 $(linux_amd64_dir)/azcopy
+              mv $(root_dir)/azcopy_linux_se_amd64 $(linux_se_amd64_dir)/azcopy
+              cp $(linux_amd64_dir)/azcopy pkgDir/usr/bin/
+              cp NOTICE.txt $(linux_amd64_dir)/
+              cp NOTICE.txt $(linux_se_amd64_dir)/
+              cp NOTICE.txt pkgDir/usr/bin/
+            displayName: 'Copy required files for packaging'
+
+          # using fpm tool for packaging of our binary & performing post-install operations
+          # for additional information about fpm refer https://fpm.readthedocs.io/en/v1.13.1/
+          - script: |
+              fpm -s dir -t deb -n azcopy --rpm-digest sha256 -C pkgDir/ \
```

For creating deb packages, you don't need `rpm-digest` flag

```suggestion
              fpm -s dir -t deb -n azcopy -C pkgDir/ \
```

---

updated